exchanges/bybit/bybit_ufutures.go

```diff
@@ -0,0 +1,1043 @@
+	updatedQty, updatedPrice, takeProfitPrice, stopLossPrice, orderTriggerPrice float64) (string, error) {
+	resp := struct {
+		Data struct {
+			OrderID string `json:"stop_order_id"`
+		} `json:"result"`
+		Error
+	}{}
+
+	params := url.Values{}
+	symbolValue, err := by.FormatSymbol(symbol, asset.USDTMarginedFutures)
+	if err != nil {
+		return "", err
+	}
+	params.Set("symbol", symbolValue)
+	if stopOrderID == "" && orderLinkID == "" {
+		return "", errStopOrderOrOrderLinkIDMissing
+	}
+	if stopOrderID != "" {
+		params.Set("stop_order_id", stopOrderID)
+	}
+	if orderLinkID != "" {
+		params.Set("order_link_id", orderLinkID)
+	}
+	if updatedQty != 0 {
+		params.Set("p_r_qty", strconv.FormatFloat(updatedQty, 'f', -1, 64))
+	}
+	if updatedPrice != 0 {
+		params.Set("p_r_price", strconv.FormatFloat(updatedPrice, 'f', -1, 64))
+	}
+	if orderTriggerPrice != 0 {
+		params.Set("p_r_trigger_price", strconv.FormatFloat(orderTriggerPrice, 'f', -1, 64))
+	}
+	if takeProfitPrice != 0 {
+		params.Set("take_profit", strconv.FormatFloat(takeProfitPrice, 'f', -1, 64))
+	}
+	if stopLossPrice != 0 {
+		params.Set("stop_loss", strconv.FormatFloat(stopLossPrice, 'f', -1, 64))
+	}
+	if takeProfitTriggerBy != "" {
+		params.Set("tp_trigger_by", takeProfitTriggerBy)
+	}
+	if stopLossTriggerBy != "" {
+		params.Set("sl_trigger_by", stopLossTriggerBy)
+	}
+	return resp.Data.OrderID, by.SendAuthHTTPRequest(ctx, exchange.RestUSDTMargined, http.MethodPost, ufuturesReplaceConditionalOrder, params, &resp, uFuturesDefaultRate)
+}
+
+// GetConditionalUSDTRealtimeOrders query real time conditional order data
+func (by *Bybit) GetConditionalUSDTRealtimeOrders(ctx context.Context, symbol currency.Pair, stopOrderID, orderLinkID string) ([]USDTFuturesConditionalRealtimeOrder, error) {
+	var data []USDTFuturesConditionalRealtimeOrder
+	params := url.Values{}
+	symbolValue, err := by.FormatSymbol(symbol, asset.USDTMarginedFutures)
+	if err != nil {
+		return data, err
+	}
+	params.Set("symbol", symbolValue)
+	if stopOrderID != "" {
+		params.Set("stop_order_id", stopOrderID)
+	}
+	if orderLinkID != "" {
+		params.Set("order_link_id", orderLinkID)
+	}
+
+	if stopOrderID == "" && orderLinkID == "" {
+		resp := struct {
+			Result []USDTFuturesConditionalRealtimeOrder `json:"result"`
+			Error
+		}{}
+		err = by.SendAuthHTTPRequest(ctx, exchange.RestUSDTMargined, http.MethodGet, ufuturesGetConditionalRealtimeOrders, params, &resp, uFuturesGetConditionalRealtimeOrderRate)
+		if err != nil {
+			return data, err
+		}
+		data = append(data, resp.Result...)
+	} else {
+		resp := struct {
+			Result USDTFuturesConditionalRealtimeOrder `json:"result"`
+			Error
+		}{}
+		err = by.SendAuthHTTPRequest(ctx, exchange.RestUSDTMargined, http.MethodGet, ufuturesGetConditionalRealtimeOrders, params, &resp, uFuturesGetConditionalRealtimeOrderRate)
+		if err != nil {
+			return data, err
+		}
+		data = append(data, resp.Result)
+	}
+	return data, nil
```

Comment:
Would you be able to update your test to also include this scenario? Even if the ID = "1234"